### PR TITLE
[wasm][debugger] Add an option to clear navigator.webdriver on attach

### PR DESCRIPTION
**Pull request by lewing**
Fixes https://github.com/mono/mono/issues/19549

### CODE REVIEW

File: sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs [hunk 1]
```diff
@@ -89,7 +92,15 @@ protected override async Task<bool> AcceptEvent (SessionId sessionId, string met
 					Log ("verbose", $"proxying Debugger.scriptParsed ({sessionId.sessionId}) {url} {args}");
 					break;
 				}
+
+			case "Target.attachedToTarget": {
+					if (args["targetInfo"]["type"]?.ToString() == "page")
+						await DeleteWebDriver (new SessionId (args["sessionId"]?.ToString ()), token);;
```
Comment (radical) [inline sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs:98]: DeleteOneSemiColon!

Comment (lewing) [reply sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs:98]: whoops!

File: sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs [hunk 2]
```diff
@@ -914,5 +936,18 @@ async Task<bool> OnGetScriptSource (MessageId msg_id, string script_id, Cancella
 			}
 			return true;
 		}
+
+		async Task DeleteWebDriver (SessionId sessionId, CancellationToken token)
```
Comment (radical) [inline sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs:940]: Maybe add a comment here referencing the issue?

**Comment by lewing** [comment]
@monojenkins build Linux x64 Bitcode+Interpreter

**Comment by lewing** [comment]
@monojenkins backport 3.2-wasm

**Pull request merged**